rest-service/manager_rest/rest/endpoint_mapper.py

```diff
@@ -99,7 +99,11 @@ def setup_resources(api):
         'ManagerConfig': 'config',
         'ManagerRestConfig': 'rest/config',
         'Agents': 'agents',
-        'AgentsName': 'agents/<string:name>'
+        'AgentsName': 'agents/<string:name>',
```

Comment:
You probably need to rebase this after my earlier commit- and allow it through the nginx config.